### ISSUE TITLE
Animation is not working in React Native with react-native-svg >= 13.0.0

### ISSUE THREAD

**Issue by alexanderoskin** [danilowoz/react-content-loader#295]
Animation is not working in React Native since react-native-svg 13.0.0

## What did you do? 
```javascript
const MyLoader = () => (
  <ContentLoader viewBox="0 0 380 70">
    <Circle cx="30" cy="30" r="30" />
    <Rect x="80" y="17" rx="4" ry="4" width="300" height="13" />
    <Rect x="80" y="40" rx="3" ry="3" width="250" height="10" />
  </ContentLoader>
)
```

## What did you expect to happen?
Loader should animate with backgroundColor and foregroundColor colors. 

## What happened actually?
No animation.

### Which versions of react-content-loader, and which browser are affected by this issue?
react-content-loader 6.2.0
react-native-svg 13.2.0
react-native 0.70.1

**Comment by BrenoPontes** [comment]
Same problem here

**Comment by thespacemanatee** [comment]
Same problem here!

**Comment by quintenbox** [comment]
Yup same problem ..

**Comment by AndreiBehel** [comment]
Confirm this issue

**Comment by TomCorvus** [comment]
@danilowoz Have we a chance to get a fix for this compatibility problem?
Thank you 🙏

**Comment by AndreiBehel** [comment]
Same problem. Is it connected with react-content-loader or react-native-svg 13?

**Comment by AlaaElden98** [comment]
I had to downgrade react-native-svg to version<=13.0.0 . But this is will not work if react-native-version is +69.
For new react-native architicture you need react-native-svg 13.0.0+ 
check https://github.com/software-mansion/react-native-svg/releases/tag/v13.0.0

**Comment by iM-GeeKy** [comment]
Is there any resolution for this that doesn't involve downgrading? I'm seeing the same behavior with the latest version of Expo. iOS seems to work fine, but Android is showing the screenshot posted by @quintenbox. Any insight would be appreciated.
```
expo 47.0.0-beta.8
react-content-loader 6.2.0
react-native-svg 13.4.0
react-native 0.70.4
```

**Comment by alexanderoskin** [comment]
The issue was in [react-native-svg](https://github.com/software-mansion/react-native-svg) and was fixed from the 13.5.0 version. So install the latest react-native-svg version. The issue can be closed.
